beamer/chain.py

```diff
@@ -382,19 +382,29 @@ def maybe_challenge(claim: Claim, context: Context) -> bool:
     if int(time.time()) < claim.challenge_back_off_timestamp:
         return False
 
-    if claim.is_challenger_winning and claim.claimer != context.address:
-        return False
-
     agent_winning = context.address in claim.get_winning_addresses()
     if agent_winning:
         return False
 
+    own_challenge_stake = Wei(claim.get_challenger_stake(context.address))
+    agent_losing = claim.claimer == context.address or own_challenge_stake > 0
```

Comment:
maybe we can find a better name for the variable

Reply:
I think `agent_participating` fits better

Reply:
Yes, I just feel this makes it harder to read the condition below. But it's just a nitpick, both are fine.